### ISSUE TITLE
MathJax support

### ISSUE THREAD

**Issue by ivanistheone** [claymcleod/pandoc-academic-publication#2]
Adding the option `--mathjax` to the commands in the Makefile enables MathJax.js rendering of math environments in the HTML versions, e.g.:

```
This is some inline math $\sin\theta$ and yada yada. Now this is a display math env:
$$
  \sin^2\theta + \cos^2 \theta = 1.
$$
```

Not sure if should be enabled by default since some papers might not have equations, but most do so I'd turn it on.

The .css for the ieee style howerver has some problems, showing equation numbering when there should be none:
<img width="386" alt="problem with eqn numbering in ieee stylesheet" src="https://cloud.githubusercontent.com/assets/163966/10417115/5fca8e92-6ffe-11e5-9769-9c1bf0e9dd59.png">

In other words, `make html-default` works great but `make html-ieee` has problems.
